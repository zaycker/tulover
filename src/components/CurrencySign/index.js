import React from 'react';
import PropTypes from 'prop-types';
import styles from './styles.module.css';
import currencySigns from 'constants/currencySigns';

export default function CurrencySign({ currency }) {
  return (
    <span className={styles['currency-sign']}>{currencySigns[currency]}</span>
  );
}

CurrencySign.propTypes = {
  currency: PropTypes.string.isRequired,
};
