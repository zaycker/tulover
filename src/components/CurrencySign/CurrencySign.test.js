import React from 'react';
import renderer from 'react-test-renderer';
import CurrencySign from './index';

it('render correctly CurrencySign component', () => {
    const CurrencySignComponent = renderer.create(<CurrencySign currency="USD" />).toJSON();
    expect(CurrencySignComponent).toMatchSnapshot();
});