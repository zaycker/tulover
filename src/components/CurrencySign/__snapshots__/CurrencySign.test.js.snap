// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly CurrencySign component 1`] = `
<span
  className="currency-sign"
>
  $
</span>
`;
